NotificationHubs/pom.xml

```diff
@@ -3,8 +3,28 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.windowsazure</groupId>
 	<artifactId>NotificationHubs</artifactId>
-	<version>0.0.4</version>
+	<version>0.0.5</version>
 	<name>Windows Azure Notification Hubs - Java REST wrapper</name>
+	<description>This package contains the JAVA SDK for Azure Notification Hubs.</description>
+	<url>https://github.com/Azure/azure-sdk-for-java</url>
+	<licenses>
+		<license>
+			<name>The MIT License (MIT)</name>
+			<url>http://opensource.org/licenses/MIT</url>
+			<distribution>repo</distribution>
+		</license>
+	</licenses>
+	<scm>
+		<url>scm:git:https://github.com/Azure/azure-sdk-for-java</url>
```

Comment:
I'm not particularly familiar with  `pom.xml`, but this line looks suspicious to me. 😃 Should the regular URL have the `scm:git:` prefix on it?